c2corg_ui/static/js/simplesearch.js

```diff
@@ -89,12 +93,21 @@ app.module.directive('appSimpleSearch', app.simpleSearchDirective);
  * @param {!angular.Scope} $scope Angular scope.
  * @param {angular.$compile} $compile Angular compile service.
  * @param {angular.$templateCache} $templateCache service
+ * @param {app.Document} appDocument service
+ * @param {app.Authentication} appAuthentication service
  * @param {angular.Attributes} $attrs Angular attributes.
  * @param {string} apiUrl Base URL of the API.
  * @param {angularGettext.Catalog} gettextCatalog Gettext catalog.
  * @ngInject
  */
-app.SimpleSearchController = function($scope, $compile, $attrs, apiUrl, gettextCatalog, $templateCache) {
+app.SimpleSearchController = function(appDocument, $scope, $compile, $attrs, apiUrl,
+                                                                            gettextCatalog, $templateCache, appAuthentication) {
```

Comment:
whoa, it looks strange on github, on my computer it's like 
![screenshot from 2016-06-02 09 07 09](https://cloud.githubusercontent.com/assets/7814311/15736660/6929b362-28a1-11e6-9bca-77d10b974f6e.png)